### ISSUE TITLE
quark change recommendations

### ISSUE THREAD

**Issue by hinhei** [Boomflex/JustNotEnoughMods2#28]
it would be appreciated to add a recipe to turn the quark chests like "quark:oak_chest"  to be able to craft those in to regular "minecraft:chest"  keep the quark chest but also give a recipe to change them to vanilla chests.

**Comment by Consumerx54** [comment]
I agree 100%  MineColonies requires the standard oak chest and will not accept others.  It was suggested to go find the regular chests in villages and mines but that is just ridiculous considering how many you need for some styles of buildings.  I spent hours and hours looking until I just gave myself a stack of regular chests

**Comment by Boomflex** [comment]
Will change in next update :)

**Comment by Xendisabu** [comment]
Just fyi, if you create a chest with more than one type of wood, you'll get the regular vanilla chest.
